src/model/AbstractPhpStruct.php

```diff
@@ -172,45 +175,63 @@ public function declareUses(string ...$uses) {
 	 * If the class has already been declared you get only the set alias.
 	 *
 	 * @param string $qualifiedName
-	 * @param null|string $alias
+	 * @param string $alias
+	 *
 	 * @return string the used alias
 	 */
-	public function declareUse(string $qualifiedName, string $alias = null) {
+	public function declareUse(string $qualifiedName, string $alias = ''): string {
 		$qualifiedName = trim($qualifiedName, '\\');
 		if (!$this->hasUseStatement($qualifiedName)) {
 			$this->addUseStatement($qualifiedName, $alias);
 		}
+
 		return $this->getUseAlias($qualifiedName);
 	}
 
 	/**
 	 * Returns whether the given use statement is present
 	 *
 	 * @param string $qualifiedName
+	 *
 	 * @return bool
 	 */
 	public function hasUseStatement(string $qualifiedName): bool {
 		return $this->useStatements->contains($qualifiedName);
 	}
 
+	public function hasUseStatementsToRender(): bool {
```

Comment:
```suggestion
	public function needsUseStatementGeneration(): bool {
```

_render_ seems a bit out of context, we want to know whether to generator them or not.